splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/LastIndexKeyOperation.java

```diff
@@ -118,6 +118,7 @@ public ExecRow getExecRowDefinition() {
         if (!isOpen)
             throw new IllegalStateException("Operation is not open");
 
+        dsp.prependSpliceExplainString(this.explainPlan);
         operationContext = dsp.createOperationContext(this);
```

Comment:
Does every operation need to call dsp.incrementOpDepth() and dsp.decrementOpDepth()? Some of them (InsertOperation, LastIndexKeyOperation, UpdateOperation, etc) don't